platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationManager.java

```diff
@@ -128,47 +142,46 @@ public void init() throws Exception {
         return export(Paths.get(exportDirectory));
     }
 
-    BundleContext getBundleContext() {
-        return FrameworkUtil.getBundle(ConfigurationMigrationManager.class)
-                .getBundleContext();
-    }
+    private Collection<MigrationWarning> exportMigratable(Migratable migratable,
+            Path exportDirectory) throws IOException {
+        Stopwatch stopwatch = null;
 
-    private Collection<MigrationWarning> exportMigratables(Path exportDirectory) {
-        List<MigrationWarning> warnings = new LinkedList<>();
+        if (LOGGER.isDebugEnabled()) {
+            stopwatch = Stopwatch.createStarted();
+        }
+
+        if (DataMigratable.class.isInstance(migratable)) {
```

Comment:
Instead of having a type check here, we should just move that code in the for loop (L181). That way the check wouldn't be needed.